### PR TITLE
Hannah Jesinkey, Todo

### PR DESCRIPTION
**Pull request by jesinkey**
Link to my deployed project: hannah-todo.netlify.app

My reflections on how this week's project turned out: Im happy to have completed the project, but again I am not happy to have had more time to work on styling and extra functionality. 

Things I'd like to have clarified or explained in more detail: I thought this week felt clear and the expectations were clear.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+import { Provider } from 'react-redux';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+
+import todos from 'reducers/todos';
+
+import TodoList from 'components/TodoList';
+import TodoForm from 'components/TodoForm';
+import TodoCounter from 'components/TodoCounter';
+import Header from 'components/Header';
+
```
Comment (malinvan) [inline code/src/App.js:11]: I like how organised your imports are :)

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <Provider store={store}>
+      <Header />
+      <TodoForm />
+      <TodoList />
+      <TodoCounter />
+    </Provider>
```
Comment (malinvan) [inline code/src/App.js:25]: Nice and clean App.js file

File: code/src/components/TodoCounter.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        {completedTodos.length}
+      </p>
+      <p className='counter'>
+        <span className='counter-text'>Total To Do's: </span>
+        {todosCounter.length}
+      </p>
+    </div>
+  );
+};
+
```
Comment (malinvan) [inline code/src/components/TodoCounter.js:24]: Think about using the styled components in future projects. At least for me it makes it easier to follow all the code needed for each component.

File: code/src/components/TodoForm.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      id: uniqid(),
+      description: value,
+      isComplete: false,
+    };
+    dispatch(todos.actions.addTodo(newTodo));
+    setValue('');
+  };
+  return (
+    <div className='form-container'>
+      <form onSubmit={onFormSubmit}>
```
Comment (malinvan) [inline code/src/components/TodoForm.js:24]: Nice and clean and easy to follow :)

**Review by malinvan — APPROVED** [review]
I really like reading your code it's very nice, clean and easy to follow. The only comment would be to maybe try using the styled components in the future. But really great job 🤩

File: code/src/components/TodoForm.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  const onFormSubmit = (e) => {
+    e.preventDefault();
+    const newTodo = {
+      id: uniqid(),
+      description: value,
+      isComplete: false,
+    };
+    dispatch(todos.actions.addTodo(newTodo));
+    setValue('');
+  };
```
Comment (KarolCislowski) [inline code/src/components/TodoForm.js:21]: you should make it impossible to send empty todo 🤓

File: code/src/components/TodoList.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+          <p className='todo-description'>{todo.description}</p>
+          <p className='date'>
+            {moment(todo.createdAt).format('dddd, D/M, k:mm')}
+          </p>
+          <button
+            className='delete-button'
+            onClick={() => dispatch(todos.actions.removeTodo(todo.id))}
+          >
+            Delete
+          </button>
```
Comment (KarolCislowski) [inline code/src/components/TodoList.js:29]: The font-weight on this background makes it nearly unreadable. The date is so small that I can't see the content.

**Review by KarolCislowski — APPROVED** [review]
Your app works very well but remember that you can make your first impression only once. Style is your first line of getting your users to try how it works.

**Pull request closed without merging**